Extension/package.json

```diff
@@ -4034,17 +4034,20 @@
       "[cpp]": {
         "editor.wordBasedSuggestions": false,
         "editor.suggest.insertMode": "replace",
-        "editor.semanticHighlighting.enabled": true
+        "editor.semanticHighlighting.enabled": true,
+        "editor.inlayHints.fontSize": 11
```

Comment:
If you have nothing (like it is right now) is it inheriting from a default in VS Code? If so the better fix would be to have that fixed at that level so that you get consistent behavior across extensions.

Reply:
Also meant to say, something that can be done separately, for now this change does what we want.

Reply:
I created a feature request on VS Code at https://github.com/microsoft/vscode/issues/152690